src/main/resources/data/events.json

```diff
@@ -25,6 +25,16 @@
     "description": "Get to work, the ship has been hit by an Asteroid!",
     "duration": 30000,
     "damage": 10
+    "parameters": [
+      {
+      	"key": "energyShield",
```

Comment:
This is a value, not a key. You should add a single parameter with key "location" and description something along the lines of "Where the malfunction is located".